src/neo-vm/Types/Array.cs

```diff
@@ -39,7 +38,7 @@ public Array(ReferenceCounter referenceCounter, IEnumerable<StackItem> value = n
             {
                 null => new List<StackItem>(),
                 List<StackItem> list => list,
-                _ => value.ToList()
+                _ => new List<StackItem>(value)
```

Comment:
I think there is no difference.

Reply:
I only want to remove `Linq` :)

Reply:
But Linq makes life better, if it doesn't affect performance.

Reply:
I think that we should receive only `List<StackItem>` instead of `IEnumerable`. It will be faster without checking types, and i think that we never use other type

Reply:
We use `IEnumerable` in many places.

https://github.com/neo-project/neo-vm/blob/66d95be5f60fdff8b1ea70fe74b9e3fe48c50684/src/neo-vm/ExecutionEngine.cs#L852

https://github.com/neo-project/neo-vm/blob/66d95be5f60fdff8b1ea70fe74b9e3fe48c50684/src/neo-vm/ExecutionEngine.cs#L859

https://github.com/neo-project/neo-vm/blob/66d95be5f60fdff8b1ea70fe74b9e3fe48c50684/src/neo-vm/ExecutionEngine.cs#L940